lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1538,9 +1545,18 @@ def cmd_sessions(*args)
         print_error("No command specified!")
         return false
       end
+      if search_term
+        matching_sessions = get_matching_sessions(search_term)
+        if matching_sessions.empty?
+          print_error("No matching sessions.") 
```

Comment:
```suggestion
          print_status("No matching sessions.") 
```
Not having sessions match your search term isn't an error, this also aligns with a later `print_status` for not matching any sessions